Extensions/3D/Model3DRuntimeObject3DRenderer.ts

```diff
@@ -144,36 +144,51 @@ namespace gdjs {
         gdjs.Model3DRuntimeObject.MaterialType.StandardWithoutMetalness
       ) {
         this._threeObject.traverse((node) => {
-          if (node.type === 'Mesh') {
-            const mesh = node as THREE.Mesh;
-            const material = mesh.material as THREE.MeshStandardMaterial;
-            //@ts-ignore
-            if (material.metalness) {
+          const mesh = node as THREE.Mesh;
+          if (!mesh.material) {
+            return;
+          }
+          const material = mesh.material;
+          if (Array.isArray(material)) {
+            for (const materialElement of material) {
               //@ts-ignore
-              material.metalness = 0;
+              if (materialElement.metalness) {
+                //@ts-ignore
+                materialElement.metalness = 0;
+              }
             }
           }
+          //@ts-ignore
+          else if (material.metalness) {
+            //@ts-ignore
+            material.metalness = 0;
+          }
         });
       } else if (
         this._model3DRuntimeObject._materialType ===
         gdjs.Model3DRuntimeObject.MaterialType.Basic
       ) {
         this._threeObject.traverse((node) => {
-          if (node.type === 'Mesh') {
-            const mesh = node as THREE.Mesh;
-            const basicMaterial = new THREE.MeshBasicMaterial();
+          const mesh = node as THREE.Mesh;
+          if (!mesh.material) {
+            return;
+          }
+          let material = mesh.material;
+          if (Array.isArray(material)) {
```

Comment:
No need to iterate on the array?

Reply:
Actually, I guess It should because according to this discussion, it can be use them on several part of the mesh:
- https://discourse.threejs.org/t/about-multiple-materials/3369/3
> Multiple materials per mesh are often used when certain parts of the geometry are rendered with different materials. A typical example are character models. In this context, [BufferGeometry.groups 306](https://threejs.org/docs/index.html#api/core/BufferGeometry.groups) defines which part of the geometry is rendered with which material.